projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -65,6 +67,12 @@ const ActionButton = ( {
 					{ __( 'Purchase', 'jetpack-my-jetpack' ) }
 				</Button>
 			);
+		case PRODUCT_STATUSES.NEEDS_PURCHASE_OR_FREE:
+			return (
+				<Button { ...buttonState } size="small" weight="regular" onClick={ onAdd }>
+					{ __( 'Try for free', 'jetpack-my-jetpack' ) }
```

Comment:
Should we use `Start for free` as everywhere else?

Reply:
Yes. I didn't realize that `Start for free` was our canonical phrase. I've reverted this change.